### DISCUSSION TITLE
Welcome to goplements.nvim Discussions!

### DISCUSSION THREAD

**Discussion by maxandron** [maxandron/goplements.nvim#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about, which are not really suitable for issue.
  * Engage with other community members.
  * Welcome others and are kind, polite, and respectful.

**Comment by marsevilspirit** [comment]
Amazing work!

**Reply by maxandron** [reply]
Thanks!
